crates/yttrium/src/chain_abstraction/api/route.rs

```diff
@@ -47,8 +47,7 @@ pub struct RouteResponseAvailable {
 #[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
 pub struct RouteResponseNotRequired {
-    #[serde(rename = "transactions")]
-    _flag: Vec<String>,
+    pub transactions: Vec<Transaction>,
```

Comment:
i understand that's initial transaction, why it's vec<> then?

Reply:
Nope it's not the initial tx, that's bridging txs, this field will be an empty array since no txs in the `RouteResponseNotRequired `.